oaknational/Oak-Web-Application
# chore: point owa to latest mv versions

pointing owa to the latest materialized views
this will resolve :
- https://www.notion.so/oaknationalacademy/Tier-on-the-lesson-overview-page-is-lower-case-79ea79f41b5444c2872e71a041c626b0?pvs=4


## How to test

1. Go to https://deploy-preview-2026--oak-web-application.netlify.thenational.academy
ensure that the user journey is as expected


## Checklist
